src/Tizen.NUI/src/internal/WebView/WebView.cs

```diff
@@ -858,6 +1167,36 @@ public void LoadHtmlString(string data)
             if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
         }
 
+        /// <summary>
+        /// Loads the specified html as the content of the view to override current history entry.
+        /// <param name="html">The html to be loaded</param>
+        /// <param name="basicUri">Basic URL used for relative paths to external objects</param>
+        /// <param name="unreachableUri">URL that could not be reached</param>
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public bool LoadHtmlStringOverrideCurrentEntry(string html, string basicUri, string unreachableUri)
```

Comment:
The term `basicUri` and `baseUri` are different? In LoadContents, `baseUri` is used but here it is `basicUri`.
I'd like to know the differences.

Reply:
Done.
basicUri is ranamed to baseUri.